wallet/src/main/scala/org/bitcoins/wallet/internal/TransactionProcessing.scala

```diff
@@ -24,6 +26,56 @@ import scala.util.{Failure, Success, Try}
   */
 private[wallet] trait TransactionProcessing extends WalletLogger {
   self: Wallet =>
+
+  private case object RebroadcastTransactionsRunnable extends Runnable {
+
+    override def run(): Unit = {
+      val f = for {
+        txs <- getTransactionsToBroadcast
+
+        _ = {
+          if (txs.size > 1)
```

Comment:
I don't really get why this is here? What does it matter if the sizes are different, aren't we logging the same thing?

Reply:
Just for the plural vs singular